/**/x500_bag_recorder_node:
 ros__parameters:
  lidar_subscriber_topic: livox/lidar
  imu_lidar_subscriber_topic: livox/imu
  vehicle_odom_subscriber_topic: fmu/out/vehicle_odometry
  rc_channels_topic: fmu/out/rc_channels
  timesync_status_topic: fmu/out/timesync_status
  file_path: /home/ciga/droneBag